custom/aaa/static/aaa/js/models/pregnant_women.js

```diff
@@ -15,17 +16,32 @@ hqDefine("aaa/js/models/pregnant_women", [
     initialPageData,
     childUtils,
     personUtils,
-    modelUtils
+    modelUtils,
+    reachUtils
 ) {
     var pregnantWomenList = function (options, postData) {
         var self = {};
         self.id = options.id;
         self.name = ko.observable(options.name);
-        self.age = ko.observable(options.age);
+        self.dob = ko.observable(options.dob);
         self.pregMonth = ko.observable(options.pregMonth);
         self.highRiskPregnancy = ko.observable(options.highRiskPregnancy);
         self.noOfAncCheckUps = ko.observable(options.noOfAncCheckUps);
 
+        self.age = ko.computed(function () {
+            if (self.dob() === 'N/A') {
+                return self.dob();
+            }
+            var age = Math.floor(moment(new Date()).diff(moment(self.dob(), "YYYY-MM-DD"),'months',true));
```

Comment:
This is using `new Date()` which will be the current date, and not the selected date. I think this also means that the tests would change values after the end of the month. I think this should be the date of the filter